src/nft/components/collection/TraitChip.tsx

```diff
@@ -0,0 +1,32 @@
+import { CrossIcon } from 'nft/components/icons'
+import styled from 'styled-components/macro'
+
+const TraitChipWrap = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px 4px 8px 12px;
+  font-weight: 600;
+  border-radius: 12px;
+  background-color: ${({ theme }) => theme.backgroundModule};
```

Comment:
should be backgroundAction

Reply:
I looked at the colors, I found this specific color under ```deprecated_bg3```.  So I made it backgroundModule instead